.changes/90312c67-354c-426d-bd15-b4f5901a3da5.json

```diff
@@ -0,0 +1,5 @@
+{
+    "id": "90312c67-354c-426d-bd15-b4f5901a3da5",
+    "type": "misc",
+    "description": "Refactor: relocate CachedCredentialsProvider and CredentialsProviderChain from aws-sdk-kotlin"
```

Comment:
**Nit**: Suggest formatting the class names (e.g., in `monospace`) for clarity when reading the changelog.